optima/model.py

```diff
@@ -110,11 +110,26 @@ def model(simpars=None, settings=None, verbose=2, safetymargin=0.8, benchmark=Fa
     numost = simpars['numost']                  # Number of people on OST (N)
     if any(injects):
         numpwid = popsize[injects,:].sum(axis=0)  # Total number of PWID
-        try: ostprev = numost/numpwid           # Proportion of PWID on OST (P)
-        except: raise OptimaException('Cannot divide by the number of PWID')
-    else:
-        if sum(numost): raise OptimaException('You have entered non-zero value for the number of PWID on OST, but you have not specified any populations who inject')
-        else: ostprev = 0.
+        try: 
+            ostprev = numost/numpwid # Proportion of PWID on OST (P)
+            ostprev = minimum(ostprev, 1.0) # Don't let more than 100% of PWID be on OST :)
```

Comment:
@cliffckerr this is ok for now but should not really be here. We could add a check like this in loadspreadsheet and notify users that way. Or we could add a check for it in parameters.py - upper limit of numost should be numpwid